exercises/space-age/.meta/generator/space_age_case.rb

```diff
@@ -0,0 +1,12 @@
+require 'generator/exercise_case'
+
+class SpaceAgeCase < Generator::ExerciseCase
+  using Generator::Underscore
```

Comment:
Yeah, I also discovered this and commented here: https://github.com/exercism/xruby/pull/583#discussion_r116457149
> It turns out you can not, due to lexical scope issues. Sorry for misleading you.
> http://blog.honeybadger.io/understanding-ruby-refinements-and-lexical-scope/

Looks like github managed to hide the comment.

I also made an issue about addressing this here in xruby: https://github.com/exercism/xruby/issues/645

But including it for now is good. 👍